app/assets/javascripts/discourse/app/templates/components/tag-groups-form.hbs

```diff
@@ -62,8 +62,7 @@
           content=allGroups
           value=selectedGroupIds
           labelProperty="name"
-          onChange=(action "setPermissionsGroups")
-        }}
```

Comment:
this is not the subject of the PR, and actually incorrect :p

Reply:
Hmm, it seemed painfully inconsistent before when compared to the rest of the file (and many other templates now that I look at it). If that change is incorrect, then we have a widespread styling problem in our templates.

Also is it improper to include minor code styling fixes in the files that you have been working with for a given PR? It seems excessive to open a separate PR, but if that's the proper thing to do, I'll definitely make note of that for the future.

Should I just revert the change to this template for now?